great-idea/css/index.css

```diff
@@ -63,4 +63,101 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Copy and paste your work from yesterday here and start to refactor into flexbox */
\ No newline at end of file
+/* Copy and paste your work from yesterday here and start to refactor into flexbox */
+.logo {
+    display: inline-block;
+    margin-left: 100px;
+}
+.container {
+    width: 1000px;
+    margin: 0 auto;
+}
+nav{
+    display: inline;
+}
+nav a{
+    text-decoration: none;
+     color: aqua;
+}
+
+.demand{
+    display: inline-block;
+    font-weight: bold;
+    margin: 0px 100px 70px 0px;
+}
+h1 {
+    font-size: 90px;
+    display: block;
+}
+.demand button{
+    display: block;
+    font-size: 20px;
+    margin-left: 60px;
+    background-color: white;
+    padding: 5px 50px;
+
+}
+.features {
+    display: inline-block;
+    width: 450px;
+    margin: 20px 20px 20px 0px;
+}
+.about {
+    display: inline-block;
+    width: 400px;
+    margin:20px 0px 20px 20px;
+}
+.main-header{
+    padding: 30px;
+    word-spacing: 50px;
+}
+.product {
+    display: inline-block;
+    width: 270px;
+    margin: 20px 20px 20px 20px;
+}
+.services {
+    display: inline-block;
+    width: 270px;
+    margin:20px 20px 20px 0px;
+}
+.vision{
+    display: inline-block;
+    width: 270px;
+    margin:20px 0px 20px 20px;
+}
+.footer{
+    text-align: center;
+    margin: 20px;
+}
+.container1{
+    display: flex;
+}
+.container2{
+    display: flex;
+}
+.container3{
+    display: flex;
+}
+.container4{
+    display: flex;
+
+}
+.box{
+    border: 1px solid black;
+    margin: 10px;
```

Comment:
As an alternative to putting margins on the boxes, you can instead put something like `justify-content: space-between;` on their container parent, and that would space the boxes apart. In fact, `space-between` will also probaby line up your boxes to be flush with the services and facts sections above and below it.